hera_qm/xrfi.py

```diff
@@ -80,43 +53,30 @@ def watershed_flag(d, f=None, sig_init=6, sig_adj=2):
     Returns:
         bool array: Array of mask values for d.
     '''
-    #mask off any points above 'sig' sigma and nan's.
-    f1 = np.ma.array(d, mask=np.where(d > sig_init,1,0)) 
+    # mask off any points above 'sig' sigma and nan's.
+    f1 = np.ma.array(d, mask=np.where(d > sig_init, 1, 0))
     f1.mask |= np.isnan(f1)
-    if not f is None: f1.mask |= f
-    
+    if not f is None:
+        f1.mask |= f
+
     # Loop over flagged points and examine adjacent points to see if they exceed sig_adj
-    #Start the watershed
-    prevx,prevy = 0,0
-    x,y = np.where(f1.mask)
+    # Start the watershed
+    prevx, prevy = 0, 0
+    x, y = np.where(f1.mask)
     while x.size != prevx and y.size != prevy:
-        for dx,dy in [(1,0),(-1,0),(0,1),(0,-1)]:
-            prevx,prevy = x.size, y.size
-            xp, yp = (x+dx).clip(0,f1.shape[0]-1), (y+dy).clip(0,f1.shape[1]-1)
-            i = np.where(f1[xp,yp] > sig_adj)[0] # if sigma > 'sigl'
-            f1.mask[xp[i],yp[i]] = 1
-            x,y = np.where(f1.mask)
+        for dx, dy in [(1, 0), (-1, 0), (0, 1), (0, -1)]:
+            prevx, prevy = x.size, y.size
+            xp, yp = (x + dx).clip(0,
+                                   f1.shape[0] - 1), (y + dy).clip(0, f1.shape[1] - 1)
+            i = np.where(f1[xp, yp] > sig_adj)[0]  # if sigma > 'sigl'
+            f1.mask[xp[i], yp[i]] = 1
+            x, y = np.where(f1.mask)
     return f1.mask
-    
-def toss_times_freqs(mask, sig_t=6, sig_f=6):
```

Comment:
Rather than delete this, I've moved it to dirty_code/misc_xrfi.py to be reviewed and a decision made.